### PR TITLE
Add new italics variant

### LINKED ISSUE #4: New italics variant?
HI @N-l1 - I really like this theme and the way it's evolving.
...so, yesterday I started playing around with it - I like a few more italics for code visual parsing, and so added a few in a variant version - not a lot, mainly HTML attribute names and JS function/method names.

If you're open to adding this, I'll create a pull request. Fully understand if you'd rather not.
Best,
Robert

### PR DESCRIPTION
**Pull request by rcrooks**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    "terminal.ansiBrightWhite": "#FFFFFF",
+    "terminal.ansiBrightYellow": "#E0CF9F",
+    "terminal.ansiCyan": "#93E0E3",
+    "terminal.ansiGreen": "#5F7F5F",
+    "terminal.ansiMagenta": "#DC8CC3",
+    "terminal.ansiRed": "#DCA3A3",
+    "terminal.ansiWhite": "#DCDCCC",
+    "terminal.ansiYellow": "#E0CF9F"
+  }
+}
```
Comment (qiz-li) [inline .vscode/settings.json]: I don't think we need the .vscode folder

File: themes/blue-cheese-italics-color-theme.json [hunk 2]
```diff
@@ -0,0 +1,1904 @@
+{
+  "name": "Blue Cheese",
```
Comment (qiz-li) [inline themes/blue-cheese-italics-color-theme.json]:
```suggestion
  "name": "Blue Cheese Italics",
```

**Review by qiz-li — CHANGES_REQUESTED** [review]
Thanks @rcrooks! They look good. Just a few small things before merging 💫

File: themes/blue-cheese-italics-color-theme.json (hunk 2 above)
Comment (rcrooks) [reply themes/blue-cheese-italics-color-theme.json]: All good - I'll fix these things and resubmit

**Comment by rcrooks** [comment]
Changed the name value for the variant and removed the .vscode folder

**Review by qiz-li — APPROVED** [review]

**Comment by qiz-li** [comment]
Thank you @rcrooks! I will draft a new release very soon.

**Pull request merged**